docs/tutorials/projects/guardian.md

```diff
@@ -0,0 +1,850 @@
+
+Next, get your Raspberry Pi and your servo and connect the servo to the Raspberry Pi by connecting the PWM wire to pin 12, the power wire to pin 2, and the ground wire to pin 8.
+Then attach the head to the servo.
+
+![A Raspberry Pi connected to a FS90R servo. The yellow PWM wire is attached to pin twelve on the raspberry pi. The red five-volt wire is attached to pin two. The black ground wire is attached to pin eight](/tutorials/img/single-component-tutorials-servo-mousemover/servo-wiring-diagram.png)
+
+Next, get the three 10mm RGB LEDs ready.
+Attach the common cathode of each LED to a [ground pin](https://pinout.xyz/pinout/ground) on your Raspberry Pi.
+Attach the wires for the red and the blue LEDs to [GPIO pins](https://pinout.xyz/pinout/wiringpi).
+
+<div class="td-max-width-on-larger-screens" style="margin: auto">
+  <img src="../../img/guardian/assembled-testing.jpg" alt="Components assembled for testing" style="max-width: 400px;" />
+</div>
+
+Before continuing with assembly, you should test your components work as expected.
+To be able to test the components, you need to install `viam-server` and configure your components.
+
+### Install `viam-server` and connect to your robot
+
+Go to the [Viam app](https://app.viam.com) and create a new robot called `guardian`.
+
+Go to the **Setup** tab of your new robot's page and follow the steps [to install `viam-server` on your computer](/installation).
+
+### Configure the components
+
+{{< tabs >}}
+{{% tab name="Builder UI" %}}
+
+Navigate to the **Config** tab of your robot's page in [the Viam app](https://app.viam.com).
+Click on the **Components** subtab and navigate to the **Create component** menu.
+
+1. **Add the board.**
+
+    Enter `local` for the name for your [board component](/components/board/), select the type `board`, and select the `pi` model.
+    Then click **Create component**.
+
+2. **Add the camera.**
+
+    Create a [camera component](/components/camera/) with the name `cam`, the type `camera` and the model `webcam`.
+    Click **Create Component** to add the camera.
+    In the new camera panel, click the **Video Path** field to reveal a drop-down populated with camera paths that have been identified on your machine.
+    Select `mmal service 16.1 (platform:bcm2835_v4l2-0)`.
+
+3. **Add the servo.**
+
+    Create a [servo component](/components/servo/) with the name `servo`, the type `servo` and the model `pi`.
+    Click **Create Component** to add the servo.
+    Configure the attributes, by adding the name of your board `local` and the pin you connected your servo to `12`:
+
+    ```json
+    {
+        "pin": "12",
+        "board": "local"
+    }
+    ```
+
+Click **Save config** in the bottom left corner of the screen.
+
+{{% /tab %}}
+{{% tab name="Raw JSON" %}}
+
+On the [`Raw JSON` tab](/manage/configuration/#the-config-tab), replace the configuration with the following configuration which configures your board, your camera, and your servo on pin `12`:
```

Comment:
```suggestion
On the [`Raw JSON` tab](/manage/configuration/#the-config-tab), replace the configuration with the following JSON configuration for your board, your camera, and your servo with its PWM wire wired to  {{< glossary_tooltip term_id="pin-number" text="pin number" >}} `12`:
```
Or something like that! I think point out where the pin number comes from helps a bit as well as the tooltip